__tests__/components/JellyForm.formulas.spec.js

```diff
@@ -49,7 +49,8 @@ describe('JellyForm formula runner should', () => {
       expect(ran).not.toEqual(data)
       expect(ran.time).toBeTruthy()
     })
-    it('when wrapped in object', () => {
+
+    it('when wrapped in an object', () => {
       const schema = {
         properties: {
           wrapper: {
```

Comment:
I know this is pre-existing, but is this supposed to be a valid JSON schema?

Reply:
good point, will go through all of them to validate

Reply:
generally yes, should be valid json schema, that includes our formula extensions

Reply:
I made sure that all the schemas render properly in rjsf